talkgenerator/util/language_util.py

```diff
@@ -238,3 +238,11 @@ def get_last_noun_and_article(sentence):
 
 def replace_non_alphabetical_characters(text):
     return re.sub(r"[^A-Za-z\s\b -]+", "", text)
+
+
+def is_vowel(word):
```

Comment:
Variable name should be `character` instead of `word` to clarify what is being tested.